packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -611,32 +327,25 @@ export abstract class QueryEditorStore {
         },
       },
     );
+    this.queryCreatorState = new QueryCreatorState(this, undefined);
   }
 
-  get isViewProjectActionDisabled(): boolean {
-    return false;
+  get label(): string | undefined {
```

Comment:
TODO: I think we can remove this altogether and in the flow of `EditExistingQueryState` we show the renamer